IT/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd very much like to but this appears to be the emerging standard for Integration Tests... thank you maven ;-)

Reply:
Not according to https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html. 

Files that match the pattern *IT.java are considered and integration test by junit, just as *Test.java are considered unit tests.

Reply:
This doc appears to not have caught up to the realities of testing Java module functionality. The naming I'm following is a practice set out by Sander Mak and is seemingly being picked up by others. My fear to not renaming is simply trying to avoid a fight with tooling. If this isn't an issue, I'm happy to changing it back to the name I originally picked, integrationtest